src/components/CustomMarker.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import {
+    Marker,
+    InfoWindow 
```

Comment:
keep consistent with line 1 - either have imports in new line or keep it all in one line